src/gateway_config.erl

```diff
@@ -58,6 +59,19 @@ mac_address(DevicePrefixes) when is_list(DevicePrefixes) ->
             end
     end.
 
+ip_address() ->
+    {ok, Addrs} = inet:getifaddrs(),
+    NonLocals = [
+        Addr || {_, Opts} <- Addrs, {addr, Addr} <- Opts,
+        size(Addr) == 4, Addr =/= {127,0,0,1}
+    ],
+    case NonLocals of
+        [] ->
+            "none";
+        _ ->
+            inet:ntoa(hd(NonLocals))
```

Comment:
What if both eth0 and wifi have an address?

Reply:
@Vagabond Like this? ```[{"wlan0_ip", "192.168.50.26"}, {"eth0_ip", "none"}]```